<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpViewer</class>
 <widget class="QWidget" name="HelpViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CLIPS Help</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTabWidget" name="tbwHelp">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabContent">
       <attribute name="title">
        <string>Content</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="lytContent"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabIndex">
       <attribute name="title">
        <string>Index</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="lytIndex"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTextBrowser" name="tebHelp">
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
